False positive warning for table character set in MySQL 8

./src/node_modules/ueberdb2/mysql_db.js warns`[ERROR] console - table is not configured with charset utf8mb4 -- This may lead to crashes when certain characters are pasted in pads`, but the table's character set is already utf8mb4:
```
mysql> SELECT CCSA.character_set_name FROM information_schema.`TABLES` T,information_schema.`COLLATION_CHARACTER_SET_APPLICABILITY` CCSA WHERE CCSA.collation_name = T.table_collation AND T.table_schema = 'etherpad' AND T.table_name = 'store';
+--------------------+
| CHARACTER_SET_NAME |
+--------------------+
| utf8mb4            |
+--------------------+
1 row in set (0.00 sec)
mysql> select version();
+-------------------------+
| version()               |
+-------------------------+
| 8.0.21-0ubuntu0.20.04.4 |
+-------------------------+
1 row in set (0.00 sec)
```

I found out no warning is reported in my old deployment with the same configure for MySQL
```
mysql> SELECT CCSA.character_set_name FROM information_schema.`TABLES` T,information_schema.`COLLATION_CHARACTER_SET_APPLICABILITY` CCSA WHERE CCSA.collation_name = T.table_collation AND T.table_schema = 'etherpad' AND T.table_name = 'store';
+--------------------+
| character_set_name |
+--------------------+
| utf8mb4            |
+--------------------+
1 row in set (0.02 sec)

mysql> select version();
+-----------------------------+
| version()                   |
+-----------------------------+
| 5.7.31-0ubuntu0.18.04.1-log |
+-----------------------------+
1 row in set (0.00 sec)
```

The difference is the column name's case for `character_set_name` returned. Something is changed in MySQL8: 
[what are the changes in mysql 8 result rowset case? - Stack Overflow](https://stackoverflow.com/questions/54538448/what-are-the-changes-in-mysql-8-result-rowset-case). `select` from view and table return column names in different case : [php - MySQL view column name capitalization - Stack Overflow](https://stackoverflow.com/questions/6750212/mysql-view-column-name-capitalization)

--- a/databases/mysql_db.js
+++ b/databases/mysql_db.js
@@ -93,7 +93,7 @@ exports.database.prototype.init = function(callback)
       };
     });
 
-    var tableCharSet = "SELECT CCSA.character_set_name FROM information_schema.`TABLES` T,information_schema.`COLLATION_CHARACTER_SET_APPLICABILITY` CCSA WHERE CCSA.collation_name = T.table_collation AND T.table_schema = '"+db.database+"' AND T.table_name = 'store'";
+    var tableCharSet = "SELECT CCSA.character_set_name AS character_set_name FROM information_schema.`TABLES` T,information_schema.`COLLATION_CHARACTER_SET_APPLICABILITY` CCSA WHERE CCSA.collation_name = T.table_collation AND T.table_schema = '"+db.database+"' AND T.table_name = 'store'";
     db.query(tableCharSet,function(err, result, tf){
       if (!result[0]) console.warn("Data has no character_set_name value -- This may lead to crashes when certain characters are pasted in pads");
       if (result[0] && (result[0].character_set_name !== db.charset)){

